test/docfx.Test/common/TestHelper.cs

```diff
@@ -9,56 +9,56 @@ namespace Microsoft.Docs.Build
 {
     internal static class TestHelper
     {
-        public static TheoryData<string, string> FindTestSpecs(string path)
+        public static TheoryData<TestSpec> FindTestSpecs(string path)
```

Comment:
Please help revert these changes, I intentionally use `TheoryData<string, string>` to avoid YAML deserialization on xunit test discovery